mythril/interfaces/cli.py

```diff
@@ -387,54 +511,54 @@ def get_code(disassembler: MythrilDisassembler, args: argparse.Namespace):
         )  # list of files
     else:
         exit_with_error(
```

Comment:
This error is reached when myth is ran without any args.
```
━━┫ ./myth                       
usage: myth [-h] [-v LOG_LEVEL]
            {analyze,a,disassemble,d,read-storage,leveldb-search,function-to-hash,hash-to-address,version,truffle,help}
            ...
mythril.interfaces.cli [ERROR]: No input bytecode. Please provide EVM code via -c BYTECODE, -a ADDRESS, -f BYTECODE_FILE or <SOLIDITY_FILE> 
```

Reply:
Oh! This should be added as a test.